nixos/tests/prometheus.nix

```diff
@@ -60,6 +60,13 @@ in import ./make-test-python.nix {
               }
             ];
           }
+          {
+            job_name = "kubernetes-sd";
+            kubernetes_sd_configs = [{
+              role = "node";
+              api_server = "http://127.0.0.1:8001";
```

Comment:
This is just a smoke test that the configuration is accept by nix, correct? E.g., there's nothing like a k8s cluster running here right?

Reply:
Right, I mentioned this in the PR desc. It only validates the config.

Reply:
Whoops, Sorry about that, I *did* read the PR description, but for some reason my brain didn't register it when writing this comment. Carry on!

Reply:
Added a comment to the test, to clarify further.